src/register.js

```diff
@@ -176,27 +178,49 @@ function configureProvider(conf = {}) {
   }
 }
 
-export default function register(conf = {}) {
+function configureNetwork(conf = {}){
   const provider = configureProvider(conf)
   const eth = new Eth(provider)
   const registryAddress = conf.registry || REGISTRY
   const DidReg = new EthContract(eth)(DidRegistryContract)
   const didReg = DidReg.at(registryAddress)
+  
+  return { eth, registryAddress, didReg}
+}
+
+function confNetworks(networksConf=[]){
```

Comment:
Naming is inconsistent by shortening to "confs" here compared to "configureNetwork."

Reply:
What should be the proper name?

Reply:
how about `configureNetworks` ?